libsql/src/hrana/cursor.rs

```diff
@@ -134,6 +134,8 @@ where
                 affected_row_count,
                 last_insert_rowid,
                 replication_index: None,
+                rows_read: 0,
+                rows_written: 0,
```

Comment:
Why are those always 0 here?

Reply:
Cursors for now don't implement stats because its too complicated and not needed for the requirements set out by the requested users.